app/Http/Controllers/Api/AssetModelsController.php

```diff
@@ -95,7 +96,7 @@ public function index(Request $request) : JsonResponse | array
         $order = $request->input('order') === 'asc' ? 'asc' : 'desc';
         $sort = in_array($request->input('sort'), $allowed_columns) ? $request->input('sort') : 'models.created_at';
 
-        switch ($sort) {
+        switch ($request->input('sort')) {
```

Comment:
Should we not be using the older 'default is models.created_at' sort variable above?

Reply:
No, the one above was overriding the sort.